tidb-cloud/branch-manage.md

```diff
@@ -67,6 +69,16 @@ To delete a branch, perform the following steps:
 4. Click **Delete** in the drop-down list.
 5. Confirm the deletion.
 
+## Reset a branch
+
+Reset will synchronize your branch with the latest data from its parent. This operation is irreversible. To reset a branch, perform the following steps:
```

Comment:
```suggestion

Resetting a branch synchronizes it with the latest data from its parent branch.

> **Note:**
> 
> This operation is irreversible. Before resetting a branch, make sure that you have backed up any important data.

To reset a branch, perform the following steps:
```

Reply:
> Resetting a branch synchronizes it with the latest data from its parent branch.
Resetting a branch synchronizes it with the latest data from its parent.
The parent can be a branch or a cluster, I think branch should be deleted

Reply:
Updated

> Resetting a branch synchronizes it with the latest data from its parent.